scripts/globals/items/guttler.lua

```diff
@@ -10,12 +10,13 @@ require("scripts/globals/magic");
 -- onAdditionalEffect Action
 -----------------------------------
 function onAdditionalEffect(player,target,damage)
+    
     local chance = 10;
-
+    
     if (math.random(0,99) >= chance or applyResistanceAddEffect(player,target,ELE_ICE,0) <= 0.5) then
         return 0,0,0;
-    else
-        target:addStatusEffect(EFFECT_CHOKE, 17, 0, 60);
-        return SUBEFFECT_CHOKE, MSGBASIC_ADD_EFFECT_STATUS, EFFECT_CHOKE;

```

Comment:
And deviating from every other existing script just because you felt like changing code that already worked. When I do it, I'm changing every instance trunk wide. It drives me insane to have your special unique snowflake staring back at me.